Svc/ComStub/test/ut/ComStubTester.cpp

```diff
@@ -113,7 +113,8 @@ void ComStubTester ::test_retry_async() {
     }
     // Retrying for as many times as the RETRY_LIMIT should be ok
     for (FwIndexType i = 0; i < this->component.RETRY_LIMIT; i++) {
-        invoke_to_drvAsyncSendReturnIn(0, buffers[i], Drv::ByteStreamStatus::SEND_RETRY);
+        invoke_to_drvAsyncSendReturnIn(
+            0, buffers[i], Drv::ByteStreamStatus::SEND_RETRY);  // NOLINT(clang-analyzer-security.ArrayBound)
```

Comment:
Not a huge deal since this is test code, but I don't understand why this warning is showing up:
- `i < this->component.RETRY_LIMIT` and `Fw::Buffer buffers[MAX_ITERS]` with `MAX_ITERS = RETRY_LIMIT + 1)`, and RETRY_LIMIT is marked `const`
- the same `buffer[i]` access is done a few lines above, with a longer loop (MAX_ITERS)

🤔